### PR TITLE
Feat/routes

### PR DESCRIPTION
**Pull request by Aleksa013**
#### 🤔 This is a ...

- [X] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 💡 Background and solution
image
image

_Describe the big picture of your changes here_
implements Route page
#### 🎯 Related issues

_Links to related issues_
https://github.com/rolling-scopes-school/tasks/blob/master/tasks/train-a/admin/routes.md

### CODE REVIEW

File: src/app/shared/components/routes/create-form/create-form.component.ts [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    private readonly store: Store,
+    private FB: FormBuilder,
+  ) {
+    this.stations$ = this.store.select(selectStations);
+    this.carriages$ = this.store.select(selectCarriages);
+    this.hasError$ = this.store.select(selectReasonError);
+  }
+
+  ngOnInit(): void {
+    this.stations$.subscribe((stations) => {
```
Comment (Arthurchkeee) [inline src/app/shared/components/routes/create-form/create-form.component.ts]: Don't forget about unsubscription

File: src/app/shared/components/routes/routes.component.html [hunk 2]
```diff
@@ -1 +1,105 @@
...
+  </div>
+  <mat-divider></mat-divider>
+  <div class="carriages-types">
+    <h4>Carriages Types:</h4>
+    <span *ngFor="let type of route.carriages; last as isLast">{{
+      isLast ? type : type + ' -'
+    }}</span>
+  </div>
+</div>
+<div class="pagination">
```
Comment (Arthurchkeee) [inline src/app/shared/components/routes/routes.component.html:83]: Perfect implementation of custom pagination!
Just for information, angular material has paginator https://material.angular.io/components/paginator/overview

**Review by Arthurchkeee — APPROVED** [review]
Good job! Please, fix the subscription

**Review by NataRep — APPROVED** [review]

File: src/app/shared/components/routes/create-form/create-form.component.ts (hunk 1 above)
Comment (Aleksa013) [reply src/app/shared/components/routes/create-form/create-form.component.ts]: Done

**Review by mogitrash — APPROVED** [review]
Nice job!

**Pull request merged**